core/src/v2/models.rs

```diff
@@ -221,6 +233,49 @@ pub struct License {
     pub url: Option<String>,
 }
 
+/// Security Scheme object.
+///
+/// https://github.com/OAI/OpenAPI-Specification/blob/master/versions/2.0.md#security-scheme-object
+#[derive(Clone, Debug, Default, Serialize, Deserialize)]
+pub struct SecurityScheme {
```

Comment:
Can you please make all the fields in this struct optional? Some of the fields are required only for specific schemes. Since they depend on the scheme, I'll have to enforce it elsewhere in code.